zallet/i18n/en-US/zallet.ftl

```diff
@@ -112,6 +112,12 @@ err-migrate-unknown-zcashd-option = Unknown {-zcashd} option '{$option}'
 err-failed-seed-fingerprinting =
     Zallet was unable to import invalid seed data, likely due to the seed having
     an invalid length.
+err-migrate-wallet-multi-import-disabled =
+    The `--allow-multiple-wallet-imports` flag must be provided to allow the
```

Comment:
```suggestion
    The `{-allow-multiple-wallet-imports}` flag must be provided to allow the
```
and add a corresponding term at the top of the file (to ensure the term is consistent across translations).